src/scenes/wizards_waltz.inc.c

```diff
@@ -1,37 +1,316 @@
-#include "asm/scenes/wizards_waltz/asm_080449a4.s"
+// For readability. !TODO - CHANGE/REMOVE
+#define GameInfo D_030055d0->gameInfo.wizardsWaltz
 
-#include "asm/scenes/wizards_waltz/asm_080449b4.s"
+// OAM Animations:
+extern u32 D_08932bec[]; // Animation: "wizard_fly"
+extern u32 D_08932c14[]; // Animation: "wizard_magic"
+extern u32 D_08932c8c[]; // Animation: "sparkle"
+extern u32 D_08932cbc[]; // Animation: "sprout_eaten"
+extern u32 D_08932d2c[]; // Animation: "sprout_grow"
+extern u32 D_08932e3c[]; // Animation: "girl"
+extern u32 D_08932e64[]; // Animation: "girl_happy"
+extern u32 D_08932ea4[]; // Animation: "girl_sad"
+extern u32 D_08932edc[]; // Animation: "shadow"
 
-#include "asm/scenes/wizards_waltz/asm_080449e4.s"
+// Sound Effects:
+extern const struct SequenceData s_witch_furu_seqData; // Sound for inputting without a cue.
 
-#include "asm/scenes/wizards_waltz/asm_08044a10.s"
+// Additional Data - Wizard's Waltz:
+extern u32 D_089e9f10; // GFX-related Null
+extern u32 D_089e9f14[]; // GFX Array
 
-#include "asm/scenes/wizards_waltz/asm_08044b80.s"
+// Additional Data - Global:
+extern s16 D_08936cac[]; // Sine Table
+extern s16 D_03004afc; // Input Queue(?)
 
-#include "asm/scenes/wizards_waltz/asm_08044ba8.s"
 
-#include "asm/scenes/wizards_waltz/asm_08044c04.s"
+/* WIZARD'S WALTZ */
 
-#include "asm/scenes/wizards_waltz/asm_08044e60.s"
 
-#include "asm/scenes/wizards_waltz/asm_08044e74.s"
+// [func_080449a4] GFX_LOAD Func_02
+void func_080449a4(void) {
+    func_0800c604(0);
+    func_08017578();
+}
 
+
+// [func_080449b4] GFX_LOAD Func_01
+void func_080449b4(void) {
+    u32 temp;
+
+    func_0800c604(0);
+    temp = func_08002ee0((u16) func_0800c3b8(), &D_089e9f14, 0x2000);
+    func_08005d38(temp, &func_080449a4, 0);
+}
+
+
+// [func_080449e4] GFX_LOAD Func_00
+void func_080449e4(void) {
+    u32 temp;
+
+    func_0800c604(0);
+    temp = func_080087b4((u16) func_0800c3b8(), &D_089e9f10);
+    func_08005d38(temp, &func_080449b4, 0);
+}
+
+
+// [func_08044a10] MAIN - Load
+void waltz_main_load(u32 arg0) {

```

Comment:
I would suggest using wizards_waltz_ as the prefix for functions in this file instead of just waltz_

Reply:
Since the function names were reverted, this issue is no longer relevant.